README.md

```diff
@@ -3407,6 +3407,29 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+## Non-code issues
+
+Now that your code is covered, consider some aspects that are not/
+should not be about *your* code:
+
+  <a name="non-code-issues--transport-and-packaging"></a><a name="31.1"></a>
+  - [31.1](#non-code-issues--transport-and-packaging)
+    **Transport and packaging**:
+    Your code should not be concerned with how it's gonna be packaged or
+    transported. Caring about those aspects should be left to automated
+    tools that are part of your build stack.
```

Comment:
can you elaborate on this a bit?

Reply:
It's the essence I distilled from #1640 :
* Project code shouldn't carry artifacts meant to fix my editor's behavior. That's my editor's problem, not the project's.
* Project code shouldn't carry artifacts meant to fix problems with how browsers obtain the code file on an HTTP level (i.e. neither about parsing nor the code's behavior). If decoration is required to ensure safe transport of the file, some tool in the build chain should add it.